api/v2/checluster_types.go

```diff
@@ -399,7 +399,7 @@ type Proxy struct {
 
 // PersistentVolumeClaim custom settings.
 type PVC struct {
-	// Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.
+	// Persistent Volume Claim size. To update the claim size, the storage class that provisions it must support resizing.
```

Comment:
It seems like many of these issues you're fixing show up throughout the repo.
Would it make sense to batch update them?
For example:
```
Results for "claim size, Storage class" in "che-operator"

/home/rdlugyhe/che-operator/api/v2/checluster_types.go:
  402:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.


/home/rdlugyhe/che-operator/config/crd/bases/org.eclipse.che_checlusters.yaml:
  1621:                                claim size, Storage class that provisions it must support
  2049:                                claim size, Storage class that provisions it must support


/home/rdlugyhe/che-operator/deploy/deployment/kubernetes/combined.yaml:
  1089:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.
  1447:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.


/home/rdlugyhe/che-operator/deploy/deployment/openshift/combined.yaml:
  1089:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.
  1447:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.


/home/rdlugyhe/che-operator/helmcharts/next/crds/checlusters.org.eclipse.che.CustomResourceDefinition.yaml:
  1084:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.
  1442:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.


/home/rdlugyhe/che-operator/bundle/next/eclipse-che-preview-openshift/manifests/org.eclipse.che_checlusters.yaml:
  1666:                                  the claim size, Storage class that provisions it must
  2101:                                  the claim size, Storage class that provisions it must


/home/rdlugyhe/che-operator/deploy/deployment/kubernetes/objects/checlusters.org.eclipse.che.CustomResourceDefinition.yaml:
  1084:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.
  1442:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.


/home/rdlugyhe/che-operator/deploy/deployment/openshift/objects/checlusters.org.eclipse.che.CustomResourceDefinition.yaml:
  1084:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.
  1442:  Persistent Volume Claim size. To update the claim size, Storage class that provisions it must support resize.

```

Reply:
I only work in the .go file in this particular repo. No idea what's happening in the other files. Let me ask.

Reply:
@deerskindoll 
The .go file is a source of truth. Other files are updated automatically after running `make update-dev-resources` command.
So, submit changes for `checluster_types.go` and I will do the rest.